### PR TITLE
Add docs for initial_peers config option

### PR DESCRIPTION
**Pull request by luckychess**
### Description of the Change
Pull request adds documentation for `initial_peers` config option

### Benefits
It's now clear how to use it.

### Possible Drawbacks 
Lack of word wrapping for example line.

### CODE REVIEW

File: docs/source/guides/configuration.rst [hunk 1]
```diff
@@ -83,6 +83,15 @@ Environment-specific parameters
   track a transaction if for some reason it is not updated with new rounds.
   However large values increase the average number of connected clients during
   each round.
+- ``"initial_peers`` is an optional parameter specifying list of peers a node will
+  use after startup instead of peers from genesis block. It could be useful when
+  you add a new node to the network where the most of initial peers became
+  malicious. Example:
```
Comment (muratovv) [inline docs/source/guides/configuration.rst]: Add words about JSON format

File: docs/source/guides/configuration.rst [hunk 2]
```diff
@@ -83,6 +83,15 @@ Environment-specific parameters
...
+
+  ``"initial_peers" : [{"address":"127.0.0.1:10001", "public_key":
+  "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"}]``
+
+  Typically you should not need to use this option.
```
Comment (muratovv) [inline docs/source/guides/configuration.rst]: Not sure that this sentence is required.

File: docs/source/guides/configuration.rst [hunk 3]
```diff
@@ -83,6 +83,13 @@ Environment-specific parameters
   track a transaction if for some reason it is not updated with new rounds.
   However large values increase the average number of connected clients during
   each round.
+- ``"initial_peers`` is an optional parameter specifying list of peers a node will
+  use after startup instead of peers from genesis block. It could be useful when
+  you add a new node to the network where the most of initial peers became
+  malicious. Peers list should be provided as a JSON array:
```
Comment (MBoldyrev) [inline docs/source/guides/configuration.rst]: I would prefer "may become malicious" as if we generally do not know who exactly is going to cheat.

**Comment by MBoldyrev** [comment]
also, @igor-egorov insists on placing each sentence on a separate line.

**Pull request merged**